facia/app/controllers/FaciaController.scala

```diff
@@ -352,8 +353,33 @@ trait FaciaController
 
   def renderShowMore(path: String, collectionId: String): Action[AnyContent] =
     Action.async { implicit request =>
-      frontJsonFapi.get(path, fullRequestType).flatMap {
+      frontJsonFapi.get(path, fullDCRRequestType).flatMap {
```

Comment:
Let's put this block inside `if(request.forceDCR)` and do `frontJsonFapi.get(path, fullRequestType)` in tan else statement